src/riak_ql_ddl.erl

```diff
@@ -412,11 +424,11 @@ is_compatible_operator('!=', boolean, boolean)-> true;
 is_compatible_operator(_,    boolean, boolean)-> false;
 is_compatible_operator(_,_,_)                 -> true.
 
--spec are_selections_valid(module(), [selection()], boolean()) ->
+-spec check_selections_valid(module(), [selection()], boolean()) ->
```

Comment:
The original name matches some of the other verification function names starting with `is_` for example